sporkfish/configurable.py

```diff
@@ -49,5 +49,4 @@ def from_dict(cls, d: Dict[Any, Any]) -> Any:
         Returns:
```

Comment:
[Nitpick] using a different style of docstring here (not reST) - not important, but I think in future code we can be clearer what style we want (in our imagined coding guideline)

Reply:
Yes - good catch - for next time pls